server/utils/tokenHandler.utils.ts

```diff
@@ -0,0 +1,30 @@
+import jwt from "jsonwebtoken";
+import envProvider from "./envProvider.utils";
+
+export interface IgenerageTokenInput {
+  userId: string;
+}
+
+interface ItokenHandler {
+  generageToken: (data: IgenerageTokenInput) => string;
+  verifyToken: (token: string) => jwt.JwtPayload | string;
+}
+
+const tokenHandler: ItokenHandler = {
```

Comment:
no need to create a tokenHandler object you can define both methods separately and export it from this file

Reply:
both methods is now as a  service